### DISCUSSION TITLE
Missing translations

### DISCUSSION THREAD

**Discussion by mat4444** [home-assistant/frontend#13545] [Localization] [answered]
Hello,
I checked the frontend and backend translation projects in Lokalise, but there are still a few parts in Home Assistant that don't have translation keys and remain in English.
Here is a non exhaustive list of UI parts that are not translatable at the moment:
- Side bar -> notification : content of the persistent notification **_"New Devices Discovered"_**
- Side bar -> Media: terms **_"Media source", "Camera", "Local Media", "Text to Speech"_**
- Settings -> Devices&Services -> Devices:   **_"< unknown >"_**  term for manufacturer and model
- Developer tools -> services: **_The services description and service attributes_** are not translated
- Developer tools -> statistics: The **_"Name" / "Statistic ID" / "Unit" / "Source" / "Issue"_** headers are not translated
- From a Dashboard -> Three dots ->  Edit dashboard : pop up **_"The edit UI is not available when in YAML mode."_** when in YAML mode only
- Profile: **_"backend-selected"_** for theme selection and **_"authenticator app"_**
- During Home Assistant loading: **_"Loading Data"_**

I'd like to contribute to the translation projects, but I don't have the right programming skills to actually modify the UI.
Thanks for checking this request!

**Comment by steverep** [comment, marked as answer]
I consider missing translations an issue to fix and not a feature request, so I converted this to issue #14416.  Please feel free to comment there if you need to edit your list.
